modules/ml/src/main/java/org/apache/ignite/ml/selection/scoring/metric/BinaryClassificationMetricValues.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.ml.selection.scoring.metric;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Provides access
+ */
+public class BinaryClassificationMetricValues {
+    /** True Positive (TP). */
+    private double tp;
+
+    /** True Negative (TN). */
+    private double tn;
+
+    /** False Positive (FP). */
+    private double fp;
+
+    /** False Negative (FN). */
+    private double fn;
+
+    /** Sensitivity or True Positive Rate (TPR). */
+    private double recall;
```

Comment:
I thought that you just add tp,tn,fp and fn values and calculate other metrics just-in-time by methods like accuracy() 
I realize that it counter-productive but computing of such metrics is a cheap procedure. IMHO a lot of fields are non-readable especially with setters.

Reply:
Good point, will think about that